src/fontra/views/editor/visualization-layer-definitions.js

```diff
@@ -208,9 +208,11 @@ registerVisualizationLayerDefinition({
     // collect paths: vertical metrics and alignment zones
     let pathZones = new Path2D();
     for (const [key, metric] of Object.entries(verticalMetrics)) {
-      let pathZone = new Path2D();
-      pathZone.rect(0, metric.value, glyphWidth, metric.zone);
-      pathZones.addPath(pathZone);
+      if (metric.zone != 0) {
+        let pathZone = new Path2D();
```

Comment:
This can be a const.

FWIW: If you never assign to a variable again after the initial assignment, you don't need let and const is enough. It has nothing to do with the mutability of the object.

Reply:
done.